stories/browser/src/UICellDebot/UICellDebot.tsx

```diff
@@ -30,26 +30,26 @@ function UICellDebotImpl({ image, title, caption, onPress, loading, testID }: UI
         <TouchableOpacity testID={testID} onPress={onPress}>
             <View style={styles.container} testID={testID}>
                 <UIImage source={image} style={styles.image as StyleProp<ImageStyle>} />
-                <View style={styles.textContainer}>
-                    {title ? (
-                        <UILabel
-                            role={TypographyVariants.NarrowActionFootnote}
-                            color={ColorVariants.TextPrimary}
-                            numberOfLines={caption ? 1 : 2}
-                        >
-                            {title}
-                        </UILabel>
-                    ) : null}
-                    {caption ? (
-                        <UILabel
-                            role={TypographyVariants.NarrowParagraphFootnote}
-                            color={ColorVariants.TextSecondary}
-                            numberOfLines={title ? 1 : 2}
-                        >
-                            {caption}
-                        </UILabel>
-                    ) : null}
-                </View>
+                {title ? (
+                    <UILabel
+                        role={TypographyVariants.NarrowActionFootnote}
+                        color={ColorVariants.TextPrimary}
+                        numberOfLines={caption ? 1 : 2}
+                        style={styles.textContainer}
```

Comment:
UILabel does not have `style` prop (and can be used only for internal purposes).
@savelichalex any ideas?

Reply:
Let's say it's still internal, as it's in @story :)

Reply:
What about TypeScript?)

Reply:
@BigRoma does it pass TS checks?